modules/installation/pages/client-requirements.adoc

```diff
@@ -3,8 +3,8 @@
 
 Supported operating systems for traditional and Salt clients are listed in this table.
 
-In this table, icon:check[role="green"] indicates that clients running the operating system are supported by {suse}, and icon:times[role="danger"] indicates that it is not supported.
-Fields marked as icon:question[role="gray"] are under consideration, and may or may not be supported at a later date.
+In this table, {check} indicates that clients running the operating system are supported by {suse}, and {cross} indicates that it is not supported.
+Fields marked as {question} are under consideration, and may or may not be supported at a later date.
 
 .Supported Versions and SP Levels
```

Comment:
I would change that "note" below a bit. This apply only for SUSE (maintained) Operating Systems.
But this is not true for other Operating Systems SUSE Manager support.
Native RHEL, CentOS, Oracle Linux, Ubuntu: here we only support the latest available version under general support.